gnovm/tests/file_test.go

```diff
@@ -100,3 +104,29 @@ func runFileTest(t *testing.T, path string, opts ...RunFileTestOption) {
 		t.Fatalf("got error: %v", err)
 	}
 }
+
+func TestRunFileTest(t *testing.T) {
+	// Get root location of github.com/gnolang/gno
+	goModPath, err := exec.Command("go", "env", "GOMOD").CombinedOutput()
+	require.NoError(t, err)
+	rootDir := path.Dir(string(goModPath))
```

Comment:
maybe?
```suggestion
	rootDir := filepath.Dir(string(goModPath))
```

Reply:
You're probably right, I assume that `go env GOMOD` returns a path in the expected OS format, so `filepath` is better! [c3d79a3](https://github.com/gnolang/gno/pull/1023/commits/c3d79a3c3909e67f173388a2b723e63eef86b4f0)